sysutils/lporg/Portfile

```diff
@@ -0,0 +1,224 @@
+                        lock    v1.6.0 \
+                        rmd160  e626beda6251efcf57f4f474d8a018a2e52f0299 \
+                        sha256  a8182db7c49cb5f1542621e651d31f13cd73dc5f9f2a161799728d23c7ef6b7f \
+                        size    94200 \
+                    github.com/fatih/color \
+                        lock    v1.13.0 \
+                        rmd160  0c56533948a292eb8c5181e9a88a45fbd1267bf5 \
+                        sha256  a65b114bfe507384e1660730803ffb4437c63a24dd11a5d7f61c77f048caa55f \
+                        size    10828 \
+                    github.com/erikstmartin/go-testdb \
+                        lock    8d10e4a1bae5 \
+                        rmd160  d62475ab3f82702a4bea2b4baa9dc4fc49426341 \
+                        sha256  5ee99717229efc96a998b6daf702c8d5da4f945048355cb144749a731aa3b8ca \
+                        size    8102 \
+                    github.com/denisenkom/go-mssqldb \
+                        lock    v0.11.0 \
+                        rmd160  001e9528200315588585cadc9aacde9c0a28a1c6 \
+                        sha256  84a40976607d8006b35a663fae79c66d1f2076d0e0e3f9f98255d5acd3b016ee \
+                        size    546180 \
+                    github.com/davecgh/go-spew \
+                        lock    v1.1.1 \
+                        rmd160  7c02883aa81f81aca14e13a27fdca9e7fbc136f7 \
+                        sha256  e85d6afa83e64962e0d63dd4812971eccf2b9b5445eda93f46a4406f0c177d5f \
+                        size    42171 \
+                    github.com/cpuguy83/go-md2man \
+                        lock    v2.0.1 \
+                        rmd160  74c013e19d22e56a27d9a3ad61b4bd86975036d1 \
+                        sha256  d23365bceea7382b59ca41ad868a80e34f8066785fb371b85b51ee0b65be6a21 \
+                        size    64243 \
+                    github.com/apex/log \
+                        lock    v1.9.0 \
+                        rmd160  db8d16bcf0b7ea316e152eaef95aee474984ffb9 \
+                        sha256  b098424ff812ffe08b754327356f22fc03a8053f4642a7b35f5262950523a222 \
+                        size    31993 \
+                    github.com/Netflix/go-expect \
+                        lock    c93bf25de8e8 \
+                        rmd160  a4e10dd1f05c584ae80408e373cc5b9e90581dc7 \
+                        sha256  896ec6b1f14446e88345be7dc7a8575957040b5a3729da1698ca88138e085ee2 \
+                        size    14580 \
+                    github.com/AlecAivazis/survey \
+                        lock    v2.3.2 \
+                        rmd160  4d4ffa6f9a7088f295ced08f60ba3a0bbbd39cb4 \
+                        sha256  5420c336cc99a5a92df0b24742f1217ab2e57ddeca497ca9ea689a2d2c5d4ff2 \
+                        size    1239289
+
+set rfc3339 [exec /bin/sh -c "/bin/date +%Y-%m-%dT%T%z | /usr/bin/sed 's/..$/:&/'"]
+
+build.env-append    CGO_ENABLED=1
+build.post_args     -ldflags '-s -w -X main.Version=${version} -X main.BuildTime=${rfc3339}'
```

Comment:
Although we do have some ports doing so, it's generally encouraged not to include `BuildTime`.  This is because it makes the build less reproducible; if we build the exact same port with the exact same dependencies one or more times, the build result will be slightly different because the `BuildTime` will be different each time.  So it would be good to consider removing it or setting it to always blank.